tests/installation/upgrade/remove_previously_used_repos.pm

```diff
@@ -0,0 +1,19 @@
+# SUSE's openQA tests
```

Comment:
as we don't validate any removal (as we discuss it could be done with test data in an extended test suite), to keep the current test modules name convention, `accept_previously _used_repos`.

Reply:
The default action of this module is to remove the repos, not to accept it, that would be confusing. I could rename it to just previously _used_repos otherwise.But I think the verb should reflect the action that is done, the convention is to use a verb describing what the module does, in this case, not "accept".

Reply:
my point is that there is not action from the user, defaults are not actions of user, are actions of the products and from the point of view of testing/qe we are more interested on seeing them as an user.
"remove_*" would imply that we remove something but we don't. As we want to re-use this test module for accepting anything that is there, more rows, more removal, etc, we are not validating anything, we are accepting what is in the screen, we didn't call "accept_defaults_*" because it was too long, but we have been following that format for long time, "accept_<name-of-the-page>.pm". In other words, in this case we don't have a verb, is an exception, we need to invent something else like "accept", "next", "go_forward", or something like that.

Reply:
I get the idea, but in that particular case it's really confusing. I propose accept_remove...